teuthology/describe_tests.py

```diff
@@ -207,9 +207,9 @@ def extract_info(file_name, fields):
     if not (isinstance(meta, list) and
             len(meta) == 1 and
             isinstance(meta[0], dict)):
-        print 'Error in meta format in', file_name
-        print 'Meta must be a list containing exactly one dict.'
-        print 'Meta is:', meta
+        print('Error in meta format in %s' % file_name)
```

Comment:
> i don't really understand why we need to `from __future__ import print_function` somewhere, but we can dispense with it in other places? so we have different assumptions on python versions in these two cases?

future print requires for, for `file=std.err` parameter and `end=''`. In other cases it is not required rather than you want to have same output for `print a, b` and `print(a,b)` which is different.

Reply:
> nit, you could also put

I could, but I prefer this.

Reply:
```
> python2 -c "print 'a'"
a
> python2 -c "print('a')"
a
> python3 -c "print('a')"
a
> python2 -c "print 'a', 'b'"
a b
> python2 -c "print('a', 'b')"
('a', 'b')
> python2 -c "from __future__ import print_function; print('a', 'b')"
a b
> python3 -c "print('a', 'b')"
a b
```